src/pages/JoinComplete/JoinComplete.Centerlogo.tsx

```
@@ -0,0 +1,11 @@
+
+
+export default function CenterLogo() {
+    return (
+    <div className="py-8">
+        <div className="relative w-32 h-32 mx-auto">
```

Biome (1.9.4)

[error] These CSS classes should be sorted.

Safe fix: Sort the classes.


(lint/nursery/useSortedClasses)